src/main/java/UseCase/createPlaylist.java

```diff
@@ -0,0 +1,21 @@
+package UseCase;
+import Entities.Song;
+import Entities.User;
+import Entities.Playlist;
+import InputBoundary.newPlaylistInputBoundary;
+
+public class createPlaylist implements newPlaylistInputBoundary {
+
+    /**
+     @param song will only be used if creating a nonempty playlist with one song
```

Comment:
The use of polymorphism is good here but could you please move the javadoc to the second method? Since the second method uses the song parameter, I think it makes sense to use the javadoc to describe the use of the parameter in the second method.

Reply:
> Generally looks good, but I saw theres still a 'collaborators' user list. Are we still doing collaborative playlists?
> 
> If we are, it feels like we should have an addCollaborator. Also, it looks like you can only retrieve a list of collaborators (and it isnt initialized, either)

For now I think it would be best to leave it, if we don't decide to do I can delete the collaborator option in the next push